packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -75,8 +75,19 @@ function preload(
       seen[dep] = true
       const isCss = dep.endsWith('.css')
       const allLinks = Array.from(document.querySelectorAll('link'))
```

Comment:
I think that the code segment for the CSS query I have in my PR is better for performance:
https://github.com/vitejs/vite/blob/8f68cdfb6650ca22c8128171b37036a595c414b5/packages/vite/src/node/plugins/importAnalysisBuild.ts#L76-L83